server-spi-private/src/main/java/org/keycloak/protocol/LoginProtocol.java

```diff
@@ -63,7 +63,12 @@
          * Passive authentication mode requested, user is logged in, but some other user interaction is necessary (eg. some required login actions exist or Consent approval is necessary for logged in
          * user)
          */
-        PASSIVE_INTERACTION_REQUIRED;
+        PASSIVE_INTERACTION_REQUIRED,
+        /**
+         * grant_id is unknown
+         */
+        INVALID_GRANT_ID,
```

Comment:
I'm not sure where it is called. If it is called from only services projects, it might be better to move it to service project's some "org.keycloak.protocol.oidc.grants.management" package.

Reply:
It is not use, I forgot to remove.